apps/calendar/js/ext/chai.js

```diff
@@ -1935,7 +1935,7 @@ module.exports = function (chai, _) {
    *
    * @name keys
    * @alias key
-   * @param {String...|Array} keys
```

Comment:
files inside the "ext" folder should not be edited since they will end up being overwritten when we update the dependencies (we don't maintain these files... all external libs)